aptos-move/framework/aptos-stdlib/sources/table.move

```diff
@@ -33,6 +33,14 @@ module aptos_std::table {
         &borrow_box<K, V, Box<V>>(table, key).val
     }
 
+    public fun borrow_with_default<K: copy + drop, V>(table: &Table<K, V>, key: K, default: &V): &V {
```

Comment:
Please make a quick doc comment above this function describing the difference for future readers :)

Reply:
@gregnazario added